README.md

```diff
@@ -5,6 +5,8 @@ GTK+3.
 
 <a href="https://appcenter.elementary.io/com.github.rkoesters.xkcd-gtk"><img height="51" alt="Get it on AppCenter" src="https://appcenter.elementary.io/badge.svg"/></a>
 <a href="https://flathub.org/apps/details/com.github.rkoesters.xkcd-gtk"><img height="51" alt="Download on Flathub" src="https://flathub.org/assets/badges/flathub-badge-en.svg"/></a>
+<a href="https://aur.archlinux.org/packages/xkcd-gtk/"><img height="51" alit="Availabe in AUR" 
+src="https://archlinux.org/static/logos/archlinux-logo-light-scalable.1ae4cc2e2469.svg"/></a>
```

Comment:
I'm hesitant to do any hot-linking of ArchLinux resources without permission. For example, the other two badges were obtained with this purpose in mind from:

* https://docs.elementary.io/develop/appcenter/markdown-badges
* https://flathub.org/badges

Perhaps we can just provide a text link, find some badge that is blessed by the ArchLinux AUR folks, or use some other solution where the linking to the image is permitted (https://shields.io/ seems like it might work, but I've never used that before).

Reply:
yeah , thats what i was afraid of , shields.io seems like a better idea
<img alt="AUR version" src="https://img.shields.io/aur/version/xkcd-gtk" height="51">